migrations/20171208101714_fix_locations_geometry_column.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * This migration transforms the `geometry` column of the `locations` table
+ * into a PostGIS geometry column rather than a PostgreSQL point column.
+ *
+ * WARNING: all locations are deleted before the operation.
+ */
+const { logMigration } = require('../utils/migrations');
+
+exports.up = async function(knex) {
+  logMigration(knex);
+  await fixLocationsGeometry(knex);
+};
+
+exports.down = async function(knex) {
+  logMigration(knex);
+  await unfixLocationsGeometry(knex);
+};
+
+async function fixLocationsGeometry(knex) {
+  await knex.delete().from('locations');
+
+  await knex.schema.alterTable('locations', t => {
+    t.dropColumn('geometry');
```

Comment:
It would probably work to alter the type, I just didn't think about that solution when writing it. Since I deleted all rows beforehand I wouldn't have had to worry about conversion anyway.